router/router_system.go

```diff
@@ -116,6 +116,12 @@ func postCreateServer(c *gin.Context) {
 // Updates the running configuration for this Wings instance.
 func postUpdateConfiguration(c *gin.Context) {
 	cfg := config.Get()
+
+	if cfg.IgnorePanelConfigUpdates {
+		c.Status(http.StatusNoContent)
```

Comment:
We probably want to send a different status code here so we can show a warning on the Panel saying that Wings is ignoring config updates, ideally something within the `2xx` range so it doesn't trigger a full error for older panels once we add the logic for a warning.

Reply:
should we send something like this?

```json
{
  "result": "ignored/applied"
}
```

Reply:
or maybe even just `applied: true/false`

Reply:
I'm thinking `{"applied":true/false}` is our best bet.  That's all the panel needs to know.